ksqldb-common/src/main/java/io/confluent/ksql/parser/NodeLocation.java

```diff
@@ -50,7 +127,9 @@ public static String asPrefix(final Optional<NodeLocation> location) {
 
   @Override
   public String toString() {
-    return String.format("Line: %d, Col: %d", line, charPositionInLine + 1);
+    return String.format("Line: %d, Col: %d",
```

Comment:
Should we update this to include stop locations? I imagine that may even be helpful for error messages that currently show that

if not, we should explicitly comment why we decided not to include them